apps/app/src/views/routes/NewOrder/components/OrderForm.tsx

```diff
@@ -218,7 +218,10 @@ export const OrderForm = ({
 
         if (
           await confirmWrapper('Send Order?', {
-            description: 'This order will be immediately sent to the pharmacy of choice.',
+            // determine if order is "Send to Patient" by checking if pharmacyId is set
+            description: values.pharmacyId
```

Comment:
Should we determine the message based `pharmacyId` vs however we know that its a "send to patient"? 

🤷

Reply:
That's currently how we determine if an order is being sent to the patient, the `createOrder` mutation is implicitly missing `pharmacyId`